app/src/ui/merge-branch/merge.tsx

```diff
@@ -258,16 +257,12 @@ export class Merge extends React.Component<IMergeProps, IMergeState> {
         id="merge"
         onDismissed={this.props.onDismissed}
         onSubmit={this.merge}
+        title={
+          <div className="merge-dialog-header">
+            Merge into <b>{currentBranchName}</b>
```

Comment:
Could we use `strong` instead of `b` here in order to stay consistent with the rebase dialog please?